### PR TITLE
fix(slider): adds padding inside of slider to fit the thumb

### LINKED ISSUE #1343: md-slider width increases when slide to right
<b>md-slider</b> width increases when sliding

and this <b>problem</b> is started from the <b>version 0.7.1-rc1</b>

check problem - <b><a href="http://plnkr.co/edit/hDPPg1UQZk7veyvPvNbJ?p=preview">http://plnkr.co/edit/hDPPg1UQZk7veyvPvNbJ?p=preview </a></b>

Same behavior is seen on angular material slider demo page.

### PR DESCRIPTION
**Pull request by robertmesserle**
Closes #1343

### CODE REVIEW

**Comment by robertmesserle** [comment]
@ThomasBurleson Please note that this will cause styles to change for some users.  Basically, in order for this to work, I had to add padding inside the actual slider wrapper, which causes the slider to shrink.  If this is not how we expect it to work, then comment here and we can discuss alternative solutions.

File: src/components/slider/slider.js [hunk 1]
```diff
@@ -85,9 +87,10 @@ function SliderDirective($$rAF, $window, $mdAria, $mdUtil, $mdConstant, $mdThemi
     };
 
     var isDisabledParsed = attr.ngDisabled && $parse(attr.ngDisabled);
-    var isDisabledGetter = isDisabledParsed ? 
+    var isDisabledGetter = isDisabledParsed ?
       function() { return isDisabledParsed(scope.$parent); } :
       angular.noop;
+    var wrapper = angular.element(element[0].querySelector('.md-slider-wrapper'));
```
Comment (ThomasBurleson) [inline src/components/slider/slider.js]: Why are you creating a unused `wrapper` variable?

**Comment by ThomasBurleson** [comment]
@robertmesserle - This effectively creates a left/right 24px padding around md-slider so the thumb shows properly at the start/end ranges. 

*  What about the ripple/focus indicators are those clipped? 
*  Have you tested with hsizing of the md-slider... does that still work ?

File: src/components/slider/slider.js (hunk 1 above)
Comment (robertmesserle) [inline src/components/slider/slider.js]: Good question - I used it at one point while fixing this, but apparently, didn't end up needing it.  Will remove.

**Comment by robertmesserle** [comment]
@ThomasBurleson The 24px padding is the size needed for the focus/held state.

Can you point me to an example of what you mean by hsizing of the slider?

**Comment by sunnyshah01** [comment]
@robertmesserle This is not the right fix. size of slider became smaller because of padding.

**Comment by robertmesserle** [comment]
@sunnyshah01 @ThomasBurleson Updated this PR to include a safer method that should not impact the size at all.

**Comment by ThomasBurleson** [comment]
@robertmesserle - `curl https://github.com/angular/material/pull/1391.patch | git am` failed.
Can you update your PR with a `git pull --rebase origin master` first? Thx.

**Pull request closed without merging**